dicodile/workers/dicod_worker.py

```diff
@@ -84,7 +77,8 @@ def compute_z_hat(self):
 
         diverging = False
         if flags.INTERACTIVE_PROCESSES and self.n_workers == 1:
-            import ipdb; ipdb.set_trace()  # noqa: E702
+            import ipdb
+            ipdb.set_trace()  # noqa: E702
```

Comment:
In this case, you need to remove the `noqa`
```suggestion
            ipdb.set_trace() 
```

Reply:
Sorry, it's my emacs which auto formats.